.github/workflows/cd.yml

```diff
@@ -0,0 +1,49 @@
+name: cd-front
+run-name: ${{ github.actor }} is executing cd-front
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read  # This is required for actions/checkout
+
+# run on merge since you cannot push to main directly
+on:
+  push: 
+    branches:
+      - build/cd-front
+
+defaults:
+  run:
+    working-directory: front
+
+jobs:
+  cd-webapp:
+    runs-on: ubuntu-latest
+    steps:
+      # Followed https://github.com/aws-actions/configure-aws-credentials
+      # Set up GitHub OIDC using https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services
+      - name: Configure AWS credentials 
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: eu-west-2
+          role-to-assume: arn:aws:iam::111982095069:role/github-omnilog
+          role-session-name: GitHubActions-CD-Front
+      - name: Checkout main
+        uses: actions/checkout@v3
+      - name: Setup Node 18
+        uses: actions/setup-node@v3
+        with: 
+          node-version: 18
+      - name: Create env file
+        run: |
+          pwd
+          echo "${{ secrets.FRONT_ENV_FILE }}" > .env
+      - name: Install dependencies
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: --cwd front install # yarn install
+      - name: Deploy
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: --cwd front sst deploy --profile omnilog --stage staging # yarn sst ...
```

Comment:
I'm suprised this was necessary (the `--cwd`)

Reply:
yes, apparently the borales/actions-yarn@v4 action is not picking up the default working directory :(